op-acceptance-tests/tests/interop/loadtest/interop_load_test.go

```diff
@@ -90,12 +90,15 @@ func SpamInteropTxs(t devtest.T, numInitTxs uint64, source *L2, dest *L2, superv
 		}
 		for msgs := range msgsCh {
 			for _, relayer := range relayers {
-				wg.Add(1)
+				relayWg.Add(1)
```

Comment:
Same question here: shouldn't `relayer` be captured?

Reply:
ditto